build.gradle

```diff
@@ -2,7 +2,6 @@ plugins {
     id 'io.franzbecker.gradle-lombok' version '3.1.0'
     id 'nebula.provided-base' version '3.0.3'
     id 'com.github.johnrengelman.shadow' version '5.2.0'
-    id "com.jfrog.bintray" version "1.8.4" apply false
```

Comment:
It's not quite as easy as this; we need to replace our bintray publication with something else, and I'm afraid that this is a task that core maintainers need to do.

Removal of `jcenter()` from `repositories` blocks is still valuable, though!